### DISCUSSION TITLE
plugin Zigbee for Domoticz

### DISCUSSION THREAD

**Discussion by pipiche38** [Koenkk/zigbee2mqtt#11187] [Show and tell]
@Koenkk this is just to let you know that we have opened the previously named 'ZiGate' plugin to TI CCxxx coordinator. The plugin has been renamed in 'Zigbee for Domoticz' and relys on zigpy-znp python modules.

In such you might get requests also from that side as regards to the Firmware. In that case and if the issue has not been yet qualified by us, you are more than welcome to pointed them to https://github.com/zigbeefordomoticz/Domoticz-Zigbee

Thanks for the great work you are doing, happy to collaborate with you

**Comment by Hedda** [comment]
FYI, Z-Stack ZNP firmware specific issues/questions are usually asked/answered in -> https://github.com/Koenkk/Z-Stack-firmware/issues

PS: And specific issues flashing firmware to C13xx/CC2538/CC26xx with cc2538-bsl here -> https://github.com/JelmerT/cc2538-bsl/issues or if using llama-bsl (fork of cc2538-bsl) to flash C1352/CC2538/CC2652 then refer them to -> https://github.com/electrolama/llama-bsl/
